VideoWeb/VideoWeb/ClientApp/src/sass/abstracts/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$tablet-large-breakpoint: 64em;
+$tablet-medium-breakpoint: 48em;
```

Comment:
```suggestion
$breakpoint-large: 64em;
$breakpoint-medium: 48em;
```

Reply:
Sorry I wasn't clear with the names yesterday. I think it's better to leave tablet out of them.